rotkehlchen/accounting/mixins/event.py

```diff
@@ -76,11 +76,11 @@ def process(
         ...
 
     @abstractmethod
-    def serialize(self) -> dict[str, Any]:
+    def serialize(self, to_debug: bool) -> dict[str, Any]:
```

Comment:
this is something only needed for the history events class and that made you change the rest of the code. I would create a new method and use that one, for example `serialize_basic_information` or any other name that you find better. Then you can use it where it is needed